cypress/integration/interface_value_page_test.js

```diff
@@ -20,6 +20,26 @@ describe('Interface values page tests', () => {
       });
     });
 
+    it('pressing back redirects to the device page', () => {
+      cy.fixture('device_detailed').as('device');
+      cy.fixture('test.astarte.IndividualObjectInterface').as('interface');
+      cy.fixture('test_individual_object_interface_values').as('interface_data');
```

Comment:
These three fixtures are not used. You can either remove these lines or use the fixtures to retrieve `deviceId` and `interfaceName` instead of defining them below. I am fine with both options.

Reply:
those are the fixture used in the `intercept` call. Without them the page will not load properly and display an error instead of the title/back button

Reply:
Are you sure? You are already specifying the fixture to use in each `intercept` call, that should be enough.